site/assets/scss/_search.scss

```diff
@@ -1,63 +1,84 @@
+// stylelint-disable selector-class-pattern
+
+// Boosted mod: the entire file is modified and is specific to Boosted
 .bd-search {
   position: relative;
-  display: flex; // Boosted mod
-  align-items: center; // Boosted mod
-  width: 100%;
-
-  &::after {
-    position: absolute;
-    top: .4rem;
-    right: .4rem;
-    bottom: .4rem;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    padding-right: .3125rem;
-    padding-left: .3125rem;
-    @include font-size(.75rem);
-    color: rgba($white, .65);
-    //////////////////////////////////////////////////////////////////////
-    //////////////////////////////////////////////////////////////////////
-    //////////////////////////////////////////////////////////////////////
-    // TODO: remove that comment in Bootstrap
-    //////////////////////////////////////////////////////////////////////
-    //////////////////////////////////////////////////////////////////////
-    //////////////////////////////////////////////////////////////////////
-    // content: "⌘K";
-    content: attr(data-shortcut);
-    background-color: rgba($white, .1);
-    @include border-radius(.125rem);
+  display: flex;
+  align-items: center;
+}
+
+.DocSearch-Container {
+  --docsearch-hit-shadow: none;
+
+  z-index: 1030;
+}
+
+.DocSearch-Button {
+  --docsearch-searchbox-background: #{$black};
+  --docsearch-searchbox-focus-background: #{$black};
+  --docsearch-text-color: #{$white};
+
+  margin: 0;
+  border: 0;
+
+  &:hover {
+    outline: unset;
+    box-shadow: unset;
+
+    .DocSearch-Search-Icon {
+      color: $primary;
+    }
+  }
+
+  &:focus {
```

Comment:
How don't you need to add `[data-focus-visible-added]` ? It never work when I do it like that, we will have to talk about this 😄 However, seems to work well here, maybe a bit more outline-offset ?